schemas/src/main/java/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
This file shouldn't have been included?

Reply:
The JAR file shouldn't be included, its added to GitHub releases not the codebase itself.

Reply:
Whoops that was a mistake, will remove